README.md

```diff
@@ -1,42 +1,42 @@
-# VML 
-[![Build Status](https://travis-ci.com/Crown421/VML.jl.svg?branch=master)](https://travis-ci.com/Crown421/VML.jl)
-[![Build status](https://ci.appveyor.com/api/projects/status/btdduqfsxux8fhsr?svg=true)](https://ci.appveyor.com/project/Crown421/vml-jl)
+# IntelVectorMath
+[![Build Status](https://travis-ci.com/Crown421/IntelVectorMath.jl.svg?branch=master)](https://travis-ci.com/Crown421/IntelVectorMath.jl)
+[![Build status](https://ci.appveyor.com/api/projects/status/btdduqfsxux8fhsr?svg=true)](https://ci.appveyor.com/project/Crown421/IntelVectorMath-jl)
 
 This package provides bindings to the Intel Vector Math Library for
 arithmetic and transcendental functions. Especially for large vectors it is often substantially faster than broadcasting Julia's built-in functions.
 
 ## Basic install
 
-To use VML.jl, you must have the shared libraries of the Intel Vector Math Library avilable on your system.
-The easiest option is to use [MKL.jl](https://github.com/JuliaComputing/MKL.jl) via 
+To use IntelVectorMath.jl, you must have the shared libraries of the Intel Vector Math Library avilable on your system.
+The easiest option is to use [MKL.jl](https://github.com/JuliaComputing/MKL.jl) via
 ```julia
 julia> ] add https://github.com/JuliaComputing/MKL.jl.git
 ```
 Alternatively you can install MKL directly [from intel](https://software.intel.com/en-us/mkl/choose-download).
 
 Note that intel MKL has a separate license, which you may want to check for commercial projects (see [FAQ]( https://software.intel.com/en-us/mkl/license-faq)).
 
-To install VML.jl run 
+To install IntelVectorMath.jl run
 ```julia
-julia> ] add https://github.com/JuliaMath/VML.jl
+julia> ] add https://github.com/JuliaMath/IntelVectorMath.jl
 ```
 
-## Using VML
-After loading `VML`, you have the supported function listed below available to call, i.e. `VML.sin(rand(100))`. This should provide a significant speed-up over broadcasting the Base functions.
+## Using IntelVectorMath
+After loading `IntelVectorMath`, you have the supported function listed below available to call, i.e. `IntelVectorMath.sin(rand(100))` or `IVM.sin(rand(100))` for short. This should provide a significant speed-up over broadcasting the Base functions.
```

Comment:
```suggestion
After loading `IntelVectorMath`, you have the supported function listed below available to call, i.e. `IntelVectorMath.sin(rand(100))`. For convenience the alias `IVM` is exported, so after `using` the package you can also call  `IVM.sin(rand(100))` for short. 
If you `import` the package, you can add this alias (or any other) via 
```
const IVM = IntelVectorMath
```
This should provide a significant speed-up over broadcasting the Base functions.
```

Reply:
Suggest re-ordering things slightly here? Not a huge deal:
```
After loading `IntelVectorMath`, you have the supported function listed below, 
for example `IntelVectorMath.sin(rand(100))`. 
These should provide a significant speed-up over broadcasting the Base functions.
Since the package name is quite long, an alias `IVM` is also exported to allow 
`IVM.sin(rand(100))` after `using` the package. (If you `import` the package, 
you can add this alias via `const IVM = IntelVectorMath`.)
```

Reply:
Also done